official/recommendation/ncf_keras_main.py

```diff
@@ -151,8 +151,19 @@ def _get_keras_model(params):
 
 def run_ncf(_):
   """Run NCF training and eval with Keras."""
+  # TODO(seemuch): This model does not support eval_batch_size to be
+  # different from batch_size, because in the input layers, it explicitly
+  # the batch_size to 'batch_size'
+  FLAGS.eval_batch_size = FLAGS.batch_size
```

Comment:
I think it's a little clearer if it is:
```
# TODO(seemuch): Support different train and eval batch sizes
if FLAGS.eval_batch_size != FLAGS.batch_size:
  logging.warning(
      "The Keras implementation of NCF currently does not support batch_size "
      "!= eval_batch_size ({} vs. {}). Overriding eval_batch_size to match "
      "batch_size".format(FLAGS.eval_batch_size, FLAGS.batch_size)
  )
  FLAGS.eval_batch_size = FLAGS.batch_size

params = ncf_common.parse_flags(FLAGS)

# ncf_common rounds eval_batch_size (this is needed due to a reshape during 
# eval). This carries over that rounding to batch_size as well.
params['batch_size'] = params['eval_batch_size']
```

I'll admit I didn't do a great job conveying what I was asking for.

Reply:
Done.